frameworks/Julia/oxygen/oxygen.dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM julia:latest
```

Comment:
Can you tie this down to a specific version please

Reply:
@NateBrady23 
How specific are we talking? Would setting it to version 1 be specific enough or are you looking for an exact 1.x.x version?

Reply:
If their version convention is such that breaking changes can be introduced in 1.x, then 1.x.x would be best.

Reply:
Julia is super good about backward compatibility, so setting to the latest stable version 1 should be fine.

Reply:
It's not a compatibility issue. But 1.10 does different scheduling which brings different performance. It's not good enough yet, but it's significantly better (mainly because you can avoid being blocked from precompilation)